css/project-list-pages.styl

```diff
@@ -43,6 +43,10 @@
     .pill-button
       margin-right: 10px
       padding: 1em
+      color: rgb(64,64,64)
+      font-weight: normal
+      border-color: rgb(64,64,64) !important
```

Comment:
I'm aware the use of !important here is sub-optimal, but I spent over half an hour trying to get to the bottom of why the upper nav bar has different styles in effect, and couldn't quite figure it out. This works for now so would prefer to go with this for now, and if someone who understands what is going on here a bit better wants to fix it later, they can!